pkgs/development/python-modules/flask-basicauth/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, flask
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "flask-basicauth";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jpvanhal";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-han0OjMI1XmuWKHGVpk+xZB+/+cpV1I+659zOG3hcPY=";
+  };
+
+  patches = [
+    (fetchpatch {
```

Comment:
For future maintenance, do you mind adding a quick comment why the patch is necessary, when it may be removed?

Reply:
thanks for the suggestion! i've just added a comment.